### DISCUSSION TITLE
Skip automatic forward only for Code Extension Host

### DISCUSSION THREAD

**Discussion by sevetseh28** [coder/code-server#6869] [Q&A]
Hey! I wanted to ask if anyone knows if it's possible to disable the automatic port forwarding only for certain processes. For example, I always see Code Extension Host being forwarded in the Ports panel. I'd like the rest to be automatically forwarded but this one.

Thanks!

**Comment by code-asher** [comment]
I searched the settings for `port` and I think `remote.portsAttributes` is what you are looking for.  There is an example there that shows how to disable auto forwarding for a port.

**Comment by sevetseh28** [comment]
Thank you @code-asher , I'll give that a try!
